adapter_pipelines/Optimus/adapter.wdl

```diff
@@ -139,7 +139,7 @@ workflow AdapterOptimus {
   Int max_cromwell_retries = 0
   Boolean add_md5s = false
 
-  String pipeline_tools_version = "v0.50.0"
+  String pipeline_tools_version = "se-update-schema-versions"
```

Comment:
Should this be updated after merge?

Reply:
I think it should be updated before merge

Reply:
Yes this was just for testing purposes